source/environment-variables.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Environment Variables
+description: List of environment variables that you can use with your Meteor application.
+---
+
+Here's a list of the environment variables you can provide to your application. All of these work both in development and production mode.
+
+<h2 id="BIND_IP">BIND_IP</h2>
+
+Bind the application server to a specific network interface by IP address, for example: `BIND_IP=192.168.0.2`.
+
+<h2 id="DISABLE_WEBSOCKETS">DISABLE_WEBSOCKETS</h2>
```

Comment:
This isn't a question for you, @juho, but does anyone reading this know why we use `<h2>` tags throughout the docs?  Is this to avoid slugs getting renamed?